lfs/util.go

```diff
@@ -51,12 +52,14 @@ func (f *GitFilter) CopyCallbackFile(event, filename string, index, totalFiles i
 	}
 
 	var prevWritten int64
-
+	deadline := time.Now().Add(time.Millisecond * 250)
```

Comment:
It seems to me this is similar in purpose to the `DefaultLoggingThrottle` value that's [defined](https://github.com/git-lfs/git-lfs/blob/2d8549c0348b97d04fa0ac96092a5f7167886e1a/tasklog/log.go#L18) in the `tasklog` package, which [limits](https://github.com/git-lfs/git-lfs/blob/2d8549c0348b97d04fa0ac96092a5f7167886e1a/tasklog/log.go#L211-L212) logging to once per every 200 milliseconds.

Rather than using a slightly different value here, I would suggest at least using that `tasklog.DefaultLoggingThrottle` constant both here and below.

(If we wanted to get fancy we could make this a configurable value, which is then used in both places, but for now I think it's OK to just use the same constant, as I believe we never set the `tasklog.Logger` with any other value than the [default](https://github.com/git-lfs/git-lfs/blob/2d8549c0348b97d04fa0ac96092a5f7167886e1a/tasklog/log.go#L71).)

One other thing we might consider is adding a test or two, akin to the [throttling tests](https://github.com/git-lfs/git-lfs/blob/2d8549c0348b97d04fa0ac96092a5f7167886e1a/tasklog/log_test.go#L120-L172) in the `tasklog` package.  That's a bit of a heavier lift, though.  I'm not sure we have any Go tests of the `lfs.CopyCallbackFile()` function at all, so let us know if we can help introduce some.

Reply:
I've updated the value to use the `DefaultLoggingThrottle` as the sleep duration. I've added a test for the throttle portion of this. It is not the cleanest test with a fair bit of setup and a sleep in it so if there is a better way to test it I'm all ears.